librz/bin/p/bin_elf.inc

```diff
@@ -1023,7 +1023,7 @@ static void patch_reloc_hexagon(RZ_INOUT RzBuffer *buf_patched, const ut64 patch
 // AARCH64-specific defines
 // Take the PAGE component of an address or offset.
 #define PG(x)         ((x) & ~0xFFFULL)
-#define PG_OFFSET(x)  ((x)&0xFFFULL)
+#define PG_OFFSET(x)  ((x) & 0xFFFULL)
```

Comment:
You might need to use `clang-format-16` or just revert the change.

Reply:
This was the result of running `/sys/clang-format.py`... 
I'll split the formatting changes into a separate commit

Reply:
you can download clang format 16 (precompiled) here https://github.com/muttleyxd/clang-tools-static-binaries/releases/